tmt/cli.py

```diff
@@ -1930,25 +1935,24 @@ def clean(context: Context,
     exit_code = 0
     if context.invoked_subcommand is None:
         assert context.obj.clean_logger is not None  # narrow type
-
-        # Set path to default
-        context.params['workdir_root'] = tmt.utils.WORKDIR_ROOT
+        root_path = effective_workdir_root(workdir_root)
```

Comment:
One more `root_path` we should root out and replace with `workdir_root`. We can definitely re-use the `workdir_root` name to hold the workdir root, the one we get from `effective_workdir_root`, it should be the same type and meaning.